.github/workflows/buildAndTest.yaml

```diff
@@ -140,6 +140,39 @@ jobs:
           name: Test logs - ${{ matrix.it}}
           path: "**/target/failsafe-reports/*"
 
+  releaseCheck:
+    if: |
+      github.repository == 'GoogleCloudPlatform/spring-cloud-gcp' &&
+      (
+        github.event_name == 'push' ||
+        github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
Am I reading it wrong or does it say "only do this when the pull request originated in a fork"?

Reply:
It says it's ok to run this from a user's fork.